.gitignore

```diff
@@ -0,0 +1,3 @@
+#Ignore dependencies
+node_modules
+package-lock.json
```

Comment:
This file shouldn't be ignored and be checked in instead. Also, this file has a missing newline at the end